geniesp/bpc_redcap_export_mapping.py

```diff
@@ -521,6 +521,7 @@ class BpcProjectRunner(metaclass=ABCMeta):
     _SPONSORED_PROJECT = ""
     # Redcap codes to cbioportal mapping synid and form key is in
     # version 38, 42 were last stable version(s)
+    # NOTE: Should be pointed towards latest version of table
```

Comment:
There will be a new mapping version, did we want to wait until that is in to update both at the same time?

Reply:
Yes this PR can be in place here.

Reply:
Are we waiting on MSK cBioPortal team to share the newer mapping file?

Reply:
Yes